modules/consignment-api/ecs.tf

```diff
@@ -27,6 +27,7 @@ data "template_file" "app" {
     frontend_url                 = var.frontend_url
     block_http4s                 = var.block_http4s
     block_assign_file_references = var.block_assign_file_references
+    reference_generator_url      = var.reference_generator_url
```

Comment:
Could you prefix the `local`s and `variables`s etc with `da_` when referring to the reference generator.